userspace/libsinsp/event.cpp

```diff
@@ -1039,15 +1039,15 @@ const char* sinsp_evt::get_param_as_str(uint32_t id, OUT const char** resolved_s
 					m_resolved_paramstr_storage.resize(path.length() + cwd.length() + 2, 0);
 				}
 
-				if(!sinsp_utils::concatenate_paths(&m_resolved_paramstr_storage[0],
-					(uint32_t)m_resolved_paramstr_storage.size(),
-					(char*)cwd.c_str(),
-					(uint32_t)cwd.length(),
-					path.data(),
-					path.length()))
+				if(path.length() == 0 || path[0] == '/')
```

Comment:
Nit: efficiency would suggest to use `path.empty()` here. I know, I'm being paranoid.